__tests__/app.test.js

```diff
@@ -26,17 +27,28 @@ describe('GET/api/topics', () => {
               expect(typeof topic.description).toBe("string")
           })
         })  
-      })
+      })     
+});
 
-      test('Responds with 404 if incorrect path specified', () => {
+describe('/GET incorrect path', () => {
+    test('Responds with 404 if incorrect path specified', () => {
         return request(app)
         .get('/api/notTopics')
         .expect(404)
         .then(({ body }) => {
           expect(body.msg).toBe("not found")
         })  
       })
+});
 
-      
 
+describe('/GET api', () => {
+    test('Responds information on available api endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual(endpoints)
```

Comment:
Great work requiring in the endpoint documentation here to allow for any additions / changes to the endpoints.